src/main/java/com/coravy/hudson/plugins/github/GithubProjectProperty.java

```diff
@@ -98,6 +99,7 @@ public static String displayNameFor(@Nonnull Job<?, ?> job) {
     }
 
     @Extension
+    @Symbol("githubProjectProperties")
```

Comment:
`githubProjectProperty`

Reply:
Thank you for the review. I chose "Properties" instead of "Property" because there are two values that I believe could be set: projectUrlStr and displayName. I chose the name "githubProjectProperties" instead of "githubJobProperties" because it matched more closely with the existing class name.

I'm happy to change this Symbol name to whatever the team thinks is the most appropriate. The functionality is what's important to me :) Just let me know!

Reply:
Changed symbol to 'githubProjectProperty'